dpbench/infrastructure/frameworks/dpcpp_framework.py

```diff
@@ -88,5 +88,13 @@ def _copy_from_func(usm_array):
 
     def version(self) -> str:
         """Returns the framework version."""
-        # hack the dpcpp version, need validate dpcpp available first
-        return self.info.dpcpp_version
+        import os
+        import re
+
+        version = os.popen("icpx --version").read().split("\n", 1)[0]
```

Comment:
We need exception handling here as `icpx` may not be installed.

Reply:
Also, we had a similar logic initially. However, it can lead to problems when we prepackage binaries of the SYCL extensions. The version should be what was used to build dpbench and not what it is when trying to run dpbench.

For this reason, @mingjie-intel had moved the logic into build time.